android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -270,19 +274,28 @@ public void onUrlMetadataIconReceived() {
 
   @Override
   public void onMdnsUrlFound(String url) {
-    if (!mUrlToUrlMetadata.containsKey(url)) {
-      // Fabricate the device values so that we can show these ersatz beacons
-      String mockAddress = generateMockBluetoothAddress(url.hashCode());
-      int mockRssi = 0;
-      int mockTxPower = 0;
-      mUrlToUrlMetadata.put(url, null);
-      mDeviceAddressToUrl.put(mockAddress, url);
-      mRegionResolver.onUpdate(mockAddress, mockRssi, mockTxPower);
-      MetadataResolver.findUrlMetadata(this, UriBeaconDiscoveryService.this, url, mockTxPower, mockRssi);
-    }
+    onLanUrlFound(url);
   }
 
-  private void startSearchingForUriBeacons() {
+    @Override
+    public void onSsdpUrlFound(String url) {
```

Comment:
Oh yes in my Android Studio the default setting is 4 spaces for indentation (converts tab in 4 spaces automatically). I will change it and check the code once again. BTW, to make the review easier for you just let me know what are the common formatting issues in my code in one single comment and  I will check everything again.

Thx,
Louay

From: matthew sibigtroth [mailto:notifications@github.com]
Sent: Freitag, 6. März 2015 00:05
To: google/physical-web
Cc: Bassbouss, Louay
Subject: Re: [physical-web] SSDP Support in Physical Web Android App (#291)


In android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java<https://github.com/google/physical-web/pull/291#discussion_r25911062>:

>    }

>

> -  private void startSearchingForUriBeacons() {

> +    @Override

> +    public void onSsdpUrlFound(String url) {

Please align the indentation of onSsdpUrlFound, onLanUrlFound, and startSearchingForUriBeacons to match the surrounding methods.

—
Reply to this email directly or view it on GitHub<https://github.com/google/physical-web/pull/291/files#r25911062>.